src/constants.tsx

```diff
@@ -208,6 +208,8 @@ export const ROUTINE_PLAN = 'Routine';
 export type ROUTINE_PLAN = typeof ROUTINE_PLAN;
 export const CASE_TRIGGERED_PLAN = 'Case-triggered';
 export type CASE_TRIGGERED_PLAN = typeof CASE_TRIGGERED_PLAN;
+export const INTERVENTION_TYPE = 'interventionType';
```

Comment:
Do you think that we can use the value from `useContextCodes` in settings.ts instead of adding a constant?